### ISSUE TITLE
add parameters to FlashMessages

### ISSUE THREAD

**Issue by JarekW** [fsi-open/admin-bundle#230]
add translation parameters to `FlashMessages` service methods, maybe this way:
```php
public function success($message, $parameters = [], $domain = 'FSiAdminBundle');
```

**Comment by norzechowicz** [comment]
It would be a BC break. 
Actually I think even domain should be removed from flash messages. Messages should be translated before passing them to FlashMessages

**Comment by szymach** [comment]
All that you achieve is that you need to pass the translator service into every class that uses the flash messages. I don't know any rule for when the messages should be translated and considering that Admin Bundle is inseparable from the translation component anyway, I think we can safely make the flash messages utilize it as well. All in all, the purpose of this service was to make our lives easier.
As for the BC break, we may simply move the `$parameters` variable after `$domain`.

**Comment by szymach** [comment]
Closed by #239
